mono/mini/TestDriver.cs

```diff
@@ -13,10 +13,40 @@ public class CategoryAttribute : Attribute
 		get; set;
 	}
 }
+public class TestDriverReporter
+{
+	private int failedTests = 0;
+	private int skippedTests = 0;
+	private int executedTests = 0;
+
+	public int FailedTests {
+		get {
+			return failedTests;
+		}
+	}
+
+	public int SkippedTests {
+		get {
+			return skippedTests;
+		}
+	}
+
+	public int ExecutedTests {
+		get {
+			return executedTests;
+		}
+	}
```

Comment:
Yeah, that code path is obviously stone cold. The thoughts above was more around the pattern, but if we always gets inlined property access in release builds both implementations will be equal.